XenAdmin/SettingsPanels/NRPEEditPage.CheckGroup.cs

```diff
@@ -53,7 +53,7 @@ public class CheckGroup
             public CheckGroup(string name, string labelName)
             {
                 Name = name;
-                NameCell = new DataGridViewTextBoxCell { Value = labelName };
+                NameCell = new DataGridViewTextBoxCell { Value = labelName, ToolTipText = string.Format(Messages.NRPE_METRIC_TOOLTIP, name)};

```

Comment:
Looks good. But suggest to add a whitespace before the last brace.

Reply:
Done.